internal/resources/metal/device/sweeper.go

```diff
@@ -46,12 +48,12 @@ func testSweepDevices(region string) error {
 		for _, d := range ds.Devices {
 			err := sweepDevice(ctx, metal, d)
 			if err != nil {
-				return fmt.Errorf("Error deleting device %s", err)
+				errs = multierror.Append(errs, fmt.Errorf("Error deleting device %s", err))
```

Comment:
We should probably be using `%w` instead of `%s`, perhaps with a separator (`:`).